src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -95,6 +93,7 @@
 		scan(invocation.getAnnotations());
 		scan(invocation.getTypeCasts());
 		scan(invocation.getTarget());
+		scan(invocation.getExecutable());
```

Comment:
And if you add this, you'll add an unnecessary import with https://github.com/INRIA/spoon/pull/596/files#diff-d6d96ce9408866e5f08058c0be741b93L87